src/components/Content.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { FilmTile } from "./FilmTile";
+
+export class Content extends React.PureComponent {
+    render() {
```

Comment:
I would suggest to use stateless component instead of classes if you don't have "state" inside. 
https://itnext.io/react-component-class-vs-stateless-component-e3797c7d23ab